lib/webdrivers/mswebdriver.rb

```diff
@@ -1,32 +1,46 @@
 # frozen_string_literal: true
 
+require 'nokogiri'
+
 module Webdrivers
   class MSWebdriver < Common
     class << self
+      attr_accessor :ignore
```

Comment:
What's the purpose of this?

Reply:
I'm not sure MS webdriver works with webdrivers. This is a toggle for turning off the gem functionality just for Edge.

Reply:
I was not sure of the use case, but I get it now after looking at #102.